.github/workflows/test.yaml

```diff
@@ -66,3 +66,9 @@ jobs:
         with:
           command: run
           args: --bin ping_pong_without_noise -- 10
+
+      - name: Run sv1-client-and-server example
+        uses: actions-rs/cargo@v1
+        with:
+          command: run
+          args: --bin client_and_server -- 10
```

Comment:
I would bump this up to 30s at least - it's odd that in 10s it's only doing this much:

```     Running `target/debug/client_and_server 10`
Server listening on: 127.0.0.1:44691
SystemTime { tv_sec: 1663600688, tv_nsec: 682836888 }-CLIENT - connected to server at 127.0.0.1:44691
SERVER - Accepting from: 127.0.0.1:50310
SERVER - message: {"id":"1663600688682940388","method":"mining.configure","parameters":[["version-rolling"],{}]}
Test Success - ran for 11 seconds```